src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  background-color: #e5e5e5;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 50px;
+}
+
+.nav__li {
+  list-style-type: none;
+  display: inline;
+}
+
+.nav__link {
+  text-decoration: none;
+  margin-left: 20px;
+  color: #000;
+  display: inline-block;
+  padding: 25px 0 15px;
+}
+
+.nav__links {
+  margin: 0 50px 0 23px;
+  display: inline-block;
+  position: relative;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+
+}
+
+.hover {
+  color: #00acdc;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+}
+
+.rectangle {
+  width: 37px;
+  height: 5px;
+  bottom: 0;
+  display: inline;
+  position: absolute;
+  flex-shrink: 0;
+  border-radius: 8px;
+  background: #00acdc;
 }
```

Comment:
The is_active class should be able to move between different navigation elements and add these styles that you defined in this class selector, but you need to change them a bit, because the width of your block that you apply this class to can change, so first transfer all these styles to is_active, and secondly, do an experiment, try to add the is_active class to some other block, for example to Tablets, and the blue bar should occupy the entire width of the block to which the class is added